app/assets/javascripts/hyrax/ga_events.js

```diff
@@ -16,7 +16,10 @@
 // });
 
 $(document).on("click", "#file_download", function (e) {
+  let ga4PropertyId = $("#ga4_property_id").val();;
```

Comment:
@paluchas I presume the two colons at the end is a (harmless) typo

Reply:
Thanks. Good spot.